themes/Delicious/foundation/colors.scss

```diff
@@ -331,6 +331,20 @@ $theme-color-palette-data: (
     500: #FFFFFF
   ),
 
+  custom:(
+    custom_gray_bg:#f8f8f8,
+    custom_gray:#666,
+    alertPrimary:#1e87f0,
+    alertSuccess:#32d296,
+    alertWarning:#faa05a,
+    alertDanger:#f0506e,
+    primaryBg:#d8eafc,
+    sucdcessBg:#edfbf6,
+    warningBg:#fff6ee,
+    dangerBg:#fef4f6,
+    
+  )
+
```

Comment:
@Farhan-emgage this is a no-no... was covered in a previous comment

Reply:
This is still a no-no.

Reply:
@Farhan-emgage TODO: Why is there still a custom colors mixin?